carpetmodSrc/carpet/commands/CommandRNG.java

```diff
@@ -65,6 +67,7 @@ public String getUsage(ICommandSender sender) {
      */
     public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
         if (!command_enabled("commandRNG", sender)) return;
+        if (args.length <=0) return;
```

Comment:
```suggestion
        if (args.length <=0) throw new WrongUsageException(getUsage(sender));
```
(don't merge via github UI, it needs an import too :P)

Reply:
Added, eventhough afterwards I noticed it's already imported for catching errors in my own additions xD